rewrite-pattern.js

```diff
@@ -83,6 +84,43 @@ const getUnicodePropertyEscapeSet = (value, isNegative) => {
 	return set.clone();
 };
 
+const getUnicodePropertyOfStringsData = (property, isNegative) => {
+	const data = getCharacterClassEmptyData();
+
+	if (isNegative) {
+		throw new Error("Cannot negate unicode sequence property");
```

Comment:
ultra nits:

- existing coding style throughout the project is to use single quotes for string literals
- "Unicode" instead of "unicode"